packages/contracts-bedrock/.env.example

```diff
@@ -8,6 +8,10 @@ PRIVATE_KEY_DEPLOYER=
 TENDERLY_PROJECT=
```

Comment:
can we check 
`/packages/contracts-bedrock/README.md` ?

Specifically that is mandatory to use foundry version:
`foundryup -C da2392e58bb8a7fefeba46b40c4df1afad8ccd22`